src/lib.rs

```diff
@@ -978,11 +992,12 @@ fn resolve_imports_match(
 
     let url = match append_specifier_to_base(resolution_result, after_prefix) {
       Ok(url) => url,
-      Err(_) => {
+      Err(e) => {
+        eprintln!("error {:#?}", e);
         return Err(ImportMapError::Other(format!(
           "Failed to resolve the specifier \"{:?}\" as its after-prefix
             portion \"{:?}\" could not be URL-parsed relative to the URL prefix
-            \"{:?}\" mapped to by the prefix \"{:?}\"",
+            \"{}\" mapped to by the prefix \"{:?}\"",
```

Comment:
```suggestion
            \"{}\" mapped to by the prefix \"{}\"",
```

I think this should also be?